app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCaptureActivity.java

```diff
@@ -164,6 +164,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     presenter.nextCalculation(true);
                 }
                 break;
+            case Constants.RQ_QR_SCANNER:
+                if(resultCode == RESULT_OK) {
+                    scanTextView.updateScanResult(data.getStringExtra(Constants.EXTRA_DATA));
+                }
```

Comment:
I would add break; just in case we add another thing here in the future we avoid strange behaviours